packages/sit-onyx/src/components/OnyxRipple/OnyxRipple.vue

```diff
@@ -0,0 +1,86 @@
+<script lang="ts" setup>
+import { computed, ref } from "vue";
+import { useRipple, type RippleConfig } from "../../composables/useRipple";
+
+const rippleTrigger = ref<HTMLElement>();
+
+const config = computed<RippleConfig>(() => ({
+  color: "var(--onyx-ripple-color, var(--onyx-color-base-primary-600))",
+  container: rippleTrigger,
+}));
+
+const { ripples, hideRipple, events } = useRipple(config);
+
+defineExpose({
+  events,
+});
+</script>
+
+<template>
+  <span ref="rippleTrigger" class="onyx-ripple">
```

Comment:
```suggestion
  <span ref="rippleTrigger" class="onyx-ripple" aria-hidden="true">
```

Should we hide the ripple for screen readers? Since its only an animation

Reply:
yes why not, although the **span** should not be shown/interpreted by any screen reader